File: w06_cs/Geometry/Rectangle.cs
using System;

namespace Geometry
{
    public class Rectangle
    {
        private double width;
        private double height;

        public Rectangle(double width, double height)
        {
            this.width = width;
            this.height = height;
        }

        public double Width
        {
            get { return width; }
            set { width = value; }
        }

        public double Height
        {
            get { return height; }
            set { height = value; }
        }

        public double Area()
        {
            return width * height;
        }

        public void PrintInfo()
        {
            Console.WriteLine($"Rectangle width={width}, height={height}, area={Area()}");
        }
    }
}

File: wo6_cpp/Geometry/Rectangle.cpp
#include <iostream>
using namespace std;

namespace Geometry {
    class Rectangle {
    private:
        double width;
        double height;

    public:
        Rectangle(double w, double h) : width(w), height(h) {}
        double getWidth() const { return width; }
        void setWidth(double w) { width = w; }
        double getHeight() const { return height; }
        void setHeight(double h) { height = h; }
        double area() const { return width * height; }

        void printInfo() const {
            cout << "Rectangle width=" << width
                 << ", height=" << height
                 << ", area=" << area() << endl;
        }
    };
}
